src/java/com/sarality/app/view/dialog/CustomViewDialogFragment.java

```diff
@@ -30,6 +32,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
   }
 
   @Override
+  public void onStart() {
```

Comment:
why onStart? Why not in render? If we are worried about render being overriden make that a public final that calls a doRender which can be overriden and then render can call ActionManager.setup